app/assets/javascripts/dashboard.js.coffee

```diff
@@ -62,6 +91,10 @@ pluralizePerson = (count) ->
 
 channelBind = (userChannel, circle) ->
   userChannel.bind 'vote', (data) ->
+    positiveCount = parseInt(document.getElementById('positive-count').innerHTML)
```

Comment:
Feels like `'positive-count'`, `'negative-count'`, `'positive_count'` and `'negative_count'` should be extracted somehow and accessible via some simple API (constants or functions, whatever). Could you suggest something?